spec/descriptions/createGroupMapping.md

```diff
@@ -0,0 +1,7 @@
+Creates a mapping between a group from the IdP (LDAP, OIDC, SAML) and an Instana group.
+
+If the IdP is configured and mappings are enabled, every time a user logs in the `key` `value` pairs for this user sent by the idp will be evaluated.
+
+If they match the mapping, the user will be assigned to the group corresponding to the `groupId`.
+
+The value for id is ignored, a new value is generated.
```

Comment:
I couldn't understand this one. Does it belong to the case "If they match the mapping" ? Is the ID mentioned here "groupId"?

and does it make sense to describe the case if "key" "value" dont match the mapping?

Reply:
The payload has a field id that is ignored. I am changing this entry to 
```
Inside the payload, the `id` for the mapping is ignored, instead, Instana generates a new id.
```